web/src/pages/Route/components/Step1/MatchingRulesView.tsx

```diff
@@ -104,6 +104,9 @@ const MatchingRulesView: React.FC<RouteModule.Step1PassProps> = ({
           case 'cookie':
             renderText = 'Cookie';
             break;
+          case 'buildin':
```

Comment:
Hi, usually we will call it `builtin`, could you please provide a link on this param?

Reply:
This is the documentation for the relevant embedded parameters: http://nginx.org/en/docs/http/ngx_http_core_module.html#variables